ui/src/app/components/load-dump/load-dump.component.ts

```diff
@@ -21,11 +21,16 @@ export class LoadDumpComponent implements OnInit {
   connectForm = new FormGroup({
     dbEngine: new FormControl('mysqldump', [Validators.required]),
     filePath: new FormControl('', [Validators.required]),
+    dialect: new FormControl('', [Validators.required]),
   })
   dbEngineList = [
     { value: 'mysqldump', displayName: 'MySQL' },
     { value: 'pg_dump', displayName: 'PostgreSQL' },
   ]
+  dialect = [
+    { value: 'google_standard_sql', displayName: 'Google Standard SQL Dialect' },
+    { value: 'postgresql', displayName: 'PostgreSQL SQL Dialect' },
+  ]
```

Comment:
Same comments as the previous: 
1. We can create this a component, we will not need to duplicate a lot of the code I think
2. Having `dialect` as a field in `IDumpConfig` looks wrong.

Reply:
Modified